plugins/noauth.js

```diff
@@ -12,7 +12,7 @@ module.exports = function (opts) {
         })
       }
     },
-    parse: function (str) {
```

Comment:
I would actually prefix this one with underscore because it might be useful. It's more useful in the implementation than it is in tests. I.e. I agree with removing unused variables in tests.